config/settings.defaults.coffee

```diff
@@ -1,5 +1,11 @@
 settings =
 	redis:
+
+		pubsub:
+			host: process.env['PUBSUB_REDIS_HOST'] or process.env['REDIS_HOST'] or "localhost"
+			port: process.env['PUBSUB_TIME_REDIS_PORT'] or process.env['REDIS_PORT'] or "6379"
```

Comment:
Is this a typo `PUBSUB_TIME_REDIS_PORT`?

Reply:
copy paste from doc updater, thanks.